buildchain/buildchain/iso.py

```diff
@@ -78,13 +78,44 @@ def task_populate_iso() -> types.TaskDict:
         'task_dep': [
             '_iso_mkdir_root',
             '_iso_render_bootstrap',
+            '_iso_add_node_manifest',
             '_iso_generate_product_txt',
             'images',
             'salt_tree',
             'packaging',
         ],
     }
 
+def task__iso_mkdir_samples() -> types.TaskDict:
+    """Create ISO_ROOT/samples"""
+    return helper.Mkdir(
+            directory=constants.ISO_ROOT/"samples", task_dep=['_iso_mkdir_root']
+    ).task
+
+
+def task__iso_add_node_manifest() -> types.TaskDict:
+    """Copy the node announcement manifest to samples."""
+    dest_generic = constants.ISO_ROOT/'samples'/'new-node.yaml'
+    dest_vagrant = constants.ISO_ROOT/'samples'/'new-node_vagrant.yaml'
+    new_node_generic = [
+            constants.ROOT/'samples'/'new-node.yaml',
+            dest_generic
+    ]
+    new_node_vagrant = [
+             constants.ROOT/'samples'/'new-node_vagrant.yaml',
+             dest_vagrant
+    ]
+    return {
+         'title': lambda task: utils.title_with_target1('GENERATE', task),
+         'actions': [
```

Comment:
when opening brace is on it own line, it more legible to have the closing one on its own line as well.

```python
         'actions': [
             (coreutils.cp_file, new_node_generic),
             (coreutils.cp_file, new_node_vagrant)
         ],
```